plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -808,6 +808,7 @@ public static ColumnStatisticsObj createMetastoreColumnStatistics(String columnN
             case SHORT:
             case INT:
             case LONG:
+            case TIMESTAMP:
```

Comment:
Grouping by relevant types (`DATE` & `TIMESTAMP`) looks better even if it's redundant in my opinion.